_includes/com-issues/com-issue-memory-process-summary.md

```diff
@@ -0,0 +1,21 @@
+## Issues
+
+* Memory issues whilst developing and testing your FeatureBase database
```

Comment:
aren't these two separate issues with two separate solutions? Any reason we're grouping them together? It's not clear which cause/solution goes back to which issue

Reply:
This is one of those issues where I need some guidance on what belongs to what because the original didn't make this particularly clear >.<

Reply:
Here is the guidance, this is two issues :D. Let's split it